org.codecoop.kjo.bookletimposer.svg

```diff
@@ -0,0 +1,891 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
It's not part of upstream tarball?

Reply:
Its part of the upstream but it whasn't in the good folder for flatpak to use it so i made this i don't know how to copy it from the upstream tarball if you can show me it can simplify the flatpak ^^